src/FastBoard.cpp

```diff
@@ -1000,6 +1036,14 @@ int FastBoard::string_size(int vertex) {
     return m_stones[m_parent[vertex]];
 }
 
+/*
+   Actual (real) liberties of the string the specified vertex position belongs to.
+   Implemented using union-find.
+   The difference is speed, pseudo liberties have O(1) adding and removal,
+   and real liberties have the inverse Ackermann function for that,
+   but in practice there is a significant speed difference.
```

Comment:
This is just wrong. There's a difference between keeping track of the liberties and keeping track of the parent, and the latter is not the speed critical part.

Reply:
removed it.